SerialPrograms/Source/PokemonSV/Inference/PokemonSV_BlueberryQuestDetector.cpp

```diff
@@ -0,0 +1,113 @@
+/*  Blueberry Quest Detector
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+
+#include "CommonFramework/ImageTypes/ImageRGB32.h"
+#include "PokemonSV/Inference/PokemonSV_BlueberryQuestReader.h"
+#include "CommonFramework/ImageTypes/ImageViewRGB32.h"
+#include "CommonFramework/VideoPipeline/VideoOverlayScopes.h"
+#include "CommonFramework/ImageTools/ImageFilter.h"
+#include "PokemonSV_BlueberryQuestDetector.h"
+
+#include <iostream>
+
+namespace PokemonAutomation{
+namespace NintendoSwitch{
+namespace PokemonSV{
+
+
+BlueberryQuestDetector::~BlueberryQuestDetector() = default;
+
+BlueberryQuestDetector::BlueberryQuestDetector(Logger& logger, Color color, Language language, QuestPosition position)
+    : m_logger(logger), m_color(color), m_language(language), m_position(position)
+{
+    switch(position){
+    case QuestPosition::FIRST:
+        m_box = ImageFloatBox(0.604, 0.218, 0.282, 0.095);
+        break;
+    case QuestPosition::SECOND:
+        m_box = ImageFloatBox(0.604, 0.395, 0.282, 0.095);
+        break;
+    case QuestPosition::THIRD:
+        m_box = ImageFloatBox(0.604, 0.572, 0.282, 0.095);
+        break;
+    case QuestPosition::FOURTH:
+        m_box = ImageFloatBox(0.604, 0.749, 0.282, 0.096);
```

Comment:
is the fourth intended to be bigger ? (difference is small anyway)

Reply:
fixed, it was unintentional